ccip-cli/src/commands/manual-exec.ts

```diff
@@ -244,6 +255,49 @@ async function manualExec(
     }
     found = true
   }
+
+  // For async chains (eg. TON), the receipt may be in a separate transaction
+  if (!found) {
```

Comment:
We need to move these to the end of `executeReport`, to abstract it away from users